client-telemetry-reporter-plugin/kafka/README.md

```diff
@@ -0,0 +1,209 @@
+to an external OTLP receiver. The OTLP receiver can be any OpenTelemetry compatible backend like
+Prometheus, Jaeger, Zipkin, etc. The plugin is implemented in Java and requires the jar to be
+added to the Kafka broker libs.
+
+The OTLP receiver for the tutorial is [otel-collector](https://opentelemetry.io/docs/collector/), which is a part of the OpenTelemetry project.
+The `otel-collector` receives the metrics from the Kafka broker and forwards them to a Prometheus
+instance for visualization.
+
+## Prerequisites
+
+- Java 17 or higher
+- [Otel Collector](#otel-collector)
+- [Prometheus](#prometheus)
+- [Client Telemetry Reporter Plugin jar](#build-the-client-telemetry-reporter-plugin)
+- [Apache Kafka Broker 3.7.0 or higher](#apache-kafka-broker)
+- [Apache Kafka Client 3.7.0 or higher](#create-a-kafka-client)
+
+### Otel Collector
+
+Pull the otel-collector docker image:
+
+```shell
+docker pull otel/opentelemetry-collector-contrib
+```
+
+#### Configuration
+
+Create a configuration file `config.yaml` for the otel-collector.
+
+Example: Below configuration defines a pipeline that receives OTLP metrics from Kafka broker on `4317 gRPC port`
+and forwards them to `Prometheus on 8889 port`. The configuration adds a namespace `kip-714` to the metrics
+which can be used to identify the source of the metrics. The configuration also enables resource to telemetry
+conversion to add labels to the metrics.
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+
+processors:
+  batch:
+    send_batch_max_size: 100
+    send_batch_size: 10
+    timeout: 10s
+
+exporters:
+  prometheus:
+    endpoint: 0.0.0.0:8889
+    namespace: kip-714
+    resource_to_telemetry_conversion:
+      enabled: true
+
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheus]
+
+```
+
+#### Start the otel-collector
+
+Replace `<local_path>` with the path to the `config.yaml` file.
+
+```shell
+docker run \
+    -p 127.0.0.1:4317:4317 -p 8889:8889 -p 8888:8888 \
+    -v <local_path>/config.yaml:/etc/otelcol-contrib/config.yaml \
+    otel/opentelemetry-collector-contrib
+ ```
+
+### Prometheus
+
+Download prometheus from [here](https://prometheus.io/download/).
+
+#### Configuration
+
+Update configuration file for prometheus `prometheus.yml` to scrape metrics from otel-collector.
+```yaml
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:8889"]
+```
+
+#### Start Prometheus
+
+```shell
+./prometheus --config.file=prometheus.yml
+```
+
+### Build the client-telemetry-reporter-plugin
+
+Clone the `tutorials` repository and build the `client-telemetry-reporter-plugin` jar.
+
+```shell
+git clone git@github.com:confluentinc/tutorials.git
+cd tutorials
+./gradlew clean :client-telemetry-reporter-plugin:kafka:build
+```
```

Comment:
With the change above that has a step to clone the repo, this would change to:

```suggestion
Build the `client-telemetry-reporter-plugin` JAR.

```shell
./gradlew clean :client-telemetry-reporter-plugin:kafka:build
```